docs/root/intro/arch_overview/websocket.rst

```diff
@@ -1,14 +1,10 @@
 .. _arch_overview_websocket:
 
-Envoy currently supports two modes of Upgrade behavior, the new generic upgrade mode, and
-the old WebSocket-only TCP proxy mode.
-
-
-New style Upgrade support
+Upgrade support
```

Comment:
Sorry I think you need a master merge here.

Reply:
Yeah, my two open PRs will conflict with each other anyway, so no worries :-P

Reply:
Sure - I'll reinstate that and remove the "new style" line, since I think the summary section and subheading makes less sense in a world with one websocket path.